src/main/java/com/nexmo/quickstart/insight/StandardInsight.java

```diff
@@ -49,8 +51,19 @@ public static void main(String[] args) throws Exception {
                 ", +" + response.getCountryPrefix() +
                 ")");
 
+        System.out.println();
         System.out.println("CARRIER INFO:");
         System.out.println("Current carrier: " + response.getCurrentCarrier().getName());
         System.out.println("Original carrier: " + response.getOriginalCarrier().getName());
+
+        if (CNAM) {
+            System.out.println();
+            System.out.println("CNAM INFO:");
+            System.out.println("Caller Name: " + response.getCallerName());
+            System.out.println("Caller Type: " + response.getCallerType());
+            System.out.println("First, Last: " + response.getFirstName() + ", " + response.getLastName());
```

Comment:
Could this be System.out.println("First name: " + response.getFirstName() + ", Last name: " + response.getLastName());